README.md

```diff
@@ -0,0 +1,39 @@
+# Initrd for Ubuntu Core
+
+This repository contains source code to build initial ramdisk used in Ubuntu Core Embedded Linux OS.
+
+# Purpose
+
+See: https://en.wikipedia.org/wiki/Initial_ramdisk
+
+# Architecture
+
+In Ubuntu Core, initrd.img is part of Kernel Snap. See [integration](#integrating-with-kernel-snap) below.
+
+In UC20, initrd is migrated from script based implementation (UC16/18) to systemd based.
+
+# Building
+
+# Testing & Debugging
+
+In order to test your unit files in the initrd shell, you need to add
+
+```
+rd.systemd
+```
+
+Example of dropping into a shell in RPi-4b with UC 20, you need to modify your cmdline.
+
+To do this, edit /run/mnt/ubuntu-seed/cmdline.txt as follows:
+
+```
+wc_otg.lpm_enable=0 console=serial0,115200 elevator=deadline rng_core.default_q
+uality=700 vt.handoff=2 quiet splash systemd.debug_shell=serial0 console=tty0 da
+ngerous systemd.mask=serial-getty@.service systemd.mask=serial-console-conf@.ser
+vice systemd.mask=console-conf@.service
```

Comment:
`dangerous rd.systemd.unit=emergency.shell`

the `systemd.mask` are irrelevant...... and are about the running system, not the initrd.